src/System.Net.Security/tests/FunctionalTests/KerberosTest.cs

```diff
@@ -0,0 +1,490 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Net.Test.Common;
+using System.Security.Authentication;
+using System.Security.Principal;
+using System.Text;
+using System.Threading.Tasks;
+
+using Xunit;
+using Xunit.Abstractions;
+
+namespace System.Net.Security.Tests
+{
+    public class KDCSetup : IDisposable
+    {
+        private const string Krb5ConfigFile = "/etc/krb5.conf";
```

Comment:
Internally, NCL discussed a different kind of design for prerequisites: the proposal was to have scripts that will be required prior to test execution. These scripts will be written similarly to UWP and other types of MSDN samples (setup / cleanup) and need to run before the tests. 

I'll work on a proposal and start a design discussion.